### DISCUSSION TITLE
Url Slider

### DISCUSSION THREAD

**Discussion by pushline** [laravel/framework#51992] [Q&A]
Is there any way to change the $window from the function `getUrlSlider` on Pagination?
By default is `$window = $onEachSide + 4;` 

But to my liking on when i use only 1 in `onEachSide` it becomes weird with extra four, so this would fix my problem:
`$window = $onEachSide + 3`

I didn't found anything related to it, so any help is appreciated

**Comment by pushline** [comment]
.

**Comment by meduzen** [comment]
Hi, I ran into the same thing, and created a [quick workaround](https://github.com/laravel/framework/discussions/39160#discussioncomment-13677110).
